docs/content/doc/installation/from-source.en-us.md

```diff
@@ -15,68 +15,97 @@ menu:
 
 # Installation from source
 
-This section will not include basic [installation instructions](https://golang.org/doc/install).
-
-**Note**: Go version 1.8 or higher is required
+You should [install go](https://golang.org/doc/install) and set up your go
+environment correctly. In particular, it is recommended to set the `$GOPATH`
+environment variable and to add the go bin directory or directories
+`${GOPATH//://bin:}/bin` to the `$PATH`. See the Go wiki entry for
+[GOPATH](https://github.com/golang/go/wiki/GOPATH).
+
+**Note**: When executing make tasks that require external tools, like
+`make misspell-check`, Gitea will automatically download and build these as
+necessary. To be able to use these you must have the `"$GOPATH/bin"` directory
+on the executable path. If you don't add the go bin directory to the
+executable path you will have to manage this yourself.
+
+**Note 2**: Go version 1.8 or higher is required, however it is recommended to
```

Comment:
ah darn, when we deprecated 1.8 it seems we missed this in documentation. Can you set this to 1.9?

Although now that 1.12 is coming out in a week or two we should probably look at deprecating 1.9 too (but that is out of scope of this PR)

Reply:
Done.